### PR TITLE
refactor(instant): Use 1.8 Instant::toString

### PR DESCRIPTION
**Pull request by LuxOrik**
https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#toString--

### CODE REVIEW

**Comment by SagaStalin** [comment]
<KGB>a4377550d6b6d5e6bf5e63595e7e0cd44c7c5f85
>refactor(instant): Use Instant toString
>
>https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#toString--

Wrap body lines at 72 characters

**Comment by satblip** [comment]
if you add a date format test, it will LGTM

**Comment by aborsu** [comment]
I concur (Je veux aussi que ce soit testé)

File: src/test/scala/logger/LoggerSpec.scala [hunk 1]
```diff
@@ -39,4 +43,30 @@ class LoggerSpec extends FlatSpec  {
...
+      log.warn("test")
+      val printed = out.toString
+      oldOut.println(printed)
+
+      val pattern = Pattern.compile("\"time\":\"20\\d\\d-[01]\\d-[0-3]\\dT[0-2]\\d:[0-5]\\d:[0-5]\\d\\.\\d\\d\\dZ\"")
+      val matcher = pattern.matcher(printed)
+
+      assert(matcher.find())
+    } finally {
+      if (out != null)
```
Comment (SagaStalin) [inline src/test/scala/logger/LoggerSpec.scala]: MAJOR Null [rule](https://sonar.sagacify.com/coding_rules#rule_key=Scalastyle%3Ascalastyle_NullChecker)

File: src/test/scala/logger/LoggerSpec.scala [hunk 2]
```diff
@@ -39,4 +43,30 @@ class LoggerSpec extends FlatSpec  {
...
+        out.close
+      if (psOut != null)
```
Comment (SagaStalin) [inline src/test/scala/logger/LoggerSpec.scala]: MAJOR Null [rule](https://sonar.sagacify.com/coding_rules#rule_key=Scalastyle%3Ascalastyle_NullChecker)

File: src/test/scala/logger/LoggerSpec.scala [hunk 3]
```diff
@@ -39,4 +43,30 @@ class LoggerSpec extends FlatSpec  {
...
+        psOut.close
+      if (oldOut != null)
```
Comment (SagaStalin) [inline src/test/scala/logger/LoggerSpec.scala]: MAJOR Null [rule](https://sonar.sagacify.com/coding_rules#rule_key=Scalastyle%3Ascalastyle_NullChecker)

**Comment by SagaStalin** [comment]
SonarQube analysis reported 3 issues
* MAJOR 3 major

Watch the comments in this conversation to review them.

File: src/test/scala/logger/LoggerSpec.scala [hunk 4]
```diff
@@ -39,4 +43,30 @@ class LoggerSpec extends FlatSpec  {
...
+        Console.setOut(oldOut)
```
Comment (aborsu) [inline src/test/scala/logger/LoggerSpec.scala]: I like that

**Comment by satblip** [comment]
LGTM

File: src/test/scala/logger/LoggerSpec.scala [hunk 5]
```diff
@@ -8,14 +13,29 @@ import Converter.map2JValue
 
 case class Poney(name: String, age: Int)
 
-class LoggerSpec extends FlatSpec  {
+class LoggerSpec extends FlatSpec with BeforeAndAfter {
+
+  def hideSTDOut(testCode: (ByteArrayOutputStream) => Any) {
```
Comment (SagaStalin) [inline src/test/scala/logger/LoggerSpec.scala]: MAJOR Use : Unit = for procedures [rule](https://sonar.sagacify.com/coding_rules#rule_key=Scalastyle%3Ascalastyle_ProcedureDeclarationChecker)

**Comment by SagaStalin** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: src/test/scala/logger/LoggerSpec.scala [hunk 6]
```diff
@@ -8,14 +12,29 @@ import Converter.map2JValue
 
 case class Poney(name: String, age: Int)
 
-class LoggerSpec extends FlatSpec  {
+class LoggerSpec extends FlatSpec {
+
+  def hideSTDOut(testCode: (ByteArrayOutputStream) => Unit) {
```
Comment (SagaStalin) [inline src/test/scala/logger/LoggerSpec.scala]: MAJOR Use : Unit = for procedures [rule](https://sonar.sagacify.com/coding_rules#rule_key=Scalastyle%3Ascalastyle_ProcedureDeclarationChecker)

**Comment by LuxOrik** [comment]
C'est beau scala

File: src/test/scala/logger/LoggerSpec.scala [hunk 7]
```diff
@@ -8,14 +12,28 @@ import Converter.map2JValue
 
 case class Poney(name: String, age: Int)
 
-class LoggerSpec extends FlatSpec  {
+class LoggerSpec extends FlatSpec {
+
+  def hideSTDOut(testCode: (ByteArrayOutputStream) => Unit): Unit = {
```
Comment (SagaStalin) [inline src/test/scala/logger/LoggerSpec.scala]: MAJOR Use : Unit = for procedures [rule](https://sonar.sagacify.com/coding_rules#rule_key=Scalastyle%3Ascalastyle_ProcedureDeclarationChecker)

**Comment by aborsu** [comment]
Amazing!!!

**Pull request merged**